src/test/java/seedu/address/testutil/TypicalAddressBook.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.testutil;
+
+import static seedu.address.testutil.TypicalProperties.getTypicalProperties;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.property.Property;
+
+
+public class TypicalAddressBook {
+    private TypicalAddressBook() {} // prevents instantiation
+
+    /**
+     * Returns an {@code AddressBook} with all the typical properties and buyers.
+     */
+    public static AddressBook getTypicalAddressBook() {
+        AddressBook ab = new AddressBook();
+        for (Property property : getTypicalProperties()) {
+            ab.addProperty(property);
+        }
+        return ab;
+    }
```

Comment:
What about buyers?

Reply:
some addressbook test breaks if I add the buyers as the JSONSerializabeleAddressBooks only contains properties. I think we can fix it in another PR.